app/src/test/java/com/android/streetworkapp/model/progression/ProgressionRepositoryFirestoreTest.kt

```diff
@@ -70,52 +69,6 @@ class ProgressionRepositoryFirestoreTest {
     assertEquals("uniqueProgressionId", progressionId)
   }
 
```

Comment:
Suggestion: There is no test on getOrAddProgression that checks behavior when the document exists or not. Not super important, if there's other pressing things to do we can leave it as is

Reply:
This PR is in fact a prelude for the actual task of this week (which is adding new features to the progression screen), therefore I think it will make more sens to make this update in the upcoming PR with all the other changes that will be made along the way.